### PR TITLE
Impl PHPDoc font-lock mechanism

### PR DESCRIPTION
**Pull request by zonuexe**
refs #327 

~~I read `cc-font.el`.  I think this change is simple and reliable.~~

I changed the implementation policy and reimplement the mechanism for font-lock.

### CODE REVIEW

**Comment by syohex** [comment]
- This change breaks test
- Could you add unit tests for this change ?

**Comment by syohex** [comment]
Could you remove `[WIP]` from title and update screenshots both current version and this version if you finish work ?

**Comment by zonuexe** [comment]
This implementation is inadequate. I will add implementation and test, so please wait a bit more.

## Current status

### Before



### After

**Comment by zonuexe** [comment]
I have finished a series of work for this PR.

File: php-mode.el [hunk 1]
```diff
@@ -603,7 +603,7 @@ but only if the setting is enabled"
 (c-add-style
  "php"
  '((c-basic-offset . 4)
-   (c-doc-comment-style . javadoc)
+   ;;(c-doc-comment-style . php-phpdoc)
```
Comment (syohex) [inline php-mode.el]: Please remove this comment line if it is not necessary

**Comment by syohex** [comment]
Thanks for great work. I'll check and merge if there are no problems.

**Comment by syohex** [comment]
Looks good

**Comment by syohex** [comment]
Thanks

**Comment by zonuexe** [comment]
Thanks!

I think https://github.com/ejmr/php-mode/commit/a6c998937341f49138f07c15050efe7e5809be23 should be released as `v1.18.0`.

**Comment by syohex** [comment]
Bump up version.

**Pull request merged**